src/new-client/src/App/Trades/TradesHeader/TradesHeader.tsx

```diff
@@ -30,17 +47,49 @@ const HeaderToolbar = styled("div")`
   align-items: center;
   justify-content: flex-end;
 `
-
-export const TradesHeader: React.FC = () => (
-  <TradesHeaderStyle>
-    <HeaderLeftGroup>Trades</HeaderLeftGroup>
-    <HeaderRightGroup>
-      <ExcelButton />
-      <HeaderToolbar>
-        <AppliedFilters />
-        <QuickFilter />
-      </HeaderToolbar>
-      <Fill />
-    </HeaderRightGroup>
-  </TradesHeaderStyle>
-)
+var isTornOut = false
+export const TradesHeader: React.FC = () => {
+  const tearOutTileState = useTearOutTileState("Tiles")
+  const tearOutAnalyticsState = useTearOutAnalyticsState("Analytics")
+  useObservableSubscription(
```

Comment:
I think you may be working off an old version of master, this util function no longer exists.